app/views/dashboard/profiles/edit.html.erb

```diff
@@ -19,7 +19,7 @@
         <%= render 'form_fields/text',
                    form: form,
                    attribute: :orcid,
-                   data: { controller: 'inputmask' } %>
+                   readonly: true %>
```

Comment:
In addition to this, we should also remove orcid from the list of permitted attributes in the controller that handles this form. We can't just rely on the front end to enforce this rule, we should also have something over on the server side too.